packages/cli/src/commands/join.ts

```diff
@@ -627,8 +625,8 @@ export async function handleJoin(argv: JoinOptions, config: Config, packageVersi
     const firstApi = documents[0];
     const openapi = firstApi.parsed;
     const componentsPrefix = getInfoPrefix(openapi.info, prefixComponentsWithInfoProp, COMPONENTS);
-    if (!openapi.openapi) exitWithError('Version of specification is not found in. \n');
-    if (!openapi.info) exitWithError('Info section is not found in specification. \n');
+    if (!openapi.openapi) exitWithError('Version of specification is not found in.');
```

Comment:
This isn't a sentence. I know we're mostly doing formatting in this PR but I can't unsee this now!

```suggestion
    if (!openapi.openapi) exitWithError('Version of specification is not found.');
```